lib/cesiumGeometryToGltfPrimitive.js

```diff
@@ -18,20 +18,32 @@ function getFirstAttributeSemantic(gltf, primitive, semantic, packedLength) {
     var type;
     if (semantic === 'TEXCOORD') {
         type = 'VEC2';
-    } else if (semantic === 'POSITION' || semantic === 'NORMAL') {
+    } else if (semantic.indexOf('POSITION') === 0 ||
+               semantic.indexOf('NORMAL')   === 0 ||
+               semantic.indexOf('TANGENT')  === 0 ||
```

Comment:
The glTF spec does not define `NORMAL` and `TANGENT` attributes.  See

https://github.com/KhronosGroup/glTF/tree/master/specification/1.0#parametersemantic

Perhaps we should add them for 1.1 since it is trivial?

Otherwise, the PBR extension could define them, but they would need to be prefixed with `_` here so the glTF asset is still valid before the extension is added.